ante/msg_filter_ante.go

```diff
@@ -0,0 +1,35 @@
+package ante
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ibcchanneltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+)
+
+// MsgFilterDecorator defines an AnteHandler decorator for the v8 upgrade that
+// provide height-gated message filtering acceptance.
+type MsgFilterDecorator struct{}
```

Comment:
Do we need to initialize it similar to how we did this in v8?

https://github.com/osmosis-labs/osmosis/blob/b82dae462c072aa6eebfa5c56ea3d22b9bb63473/app/ante.go#L39

Reply:
yes, ty